documentation/src/main/asciidoc/topics/con_persistence_spi.adoc

```diff
@@ -1,12 +1,14 @@
-[id='persistence_spi']
-:context: spi
-= {brandname} Persistence SPIs
-{brandname} Service Provider Interfaces (SPI) enable read and write operations
-to external storage and provide the following features:
+[id='persistent_spi']
+= {brandname} Persistence SPI
+The {brandname} Service Provider Interface (SPI) enables read and write
+operations to external storage through the `NonBlockingStore` interface and has
+the following features:
+
+Non-Blocking::
 
 Portability across JCache-compliant vendors::
-The {brandname} `CacheWriter` and `CacheLoader` interfaces align with the
-`JSR-107` JCache specification.
+The {brandname} `NonBlockingStore` interface aligns with the `JSR-107` JCache
```

Comment:
It doesn't really align anymore since our SPI is non blocking and theirs is blocking. However we will always have an adapter that bridges that by using the blocking thread pool.

Reply:
@wburns I wasn't sure about this one. Should we just remove it or better to say something like:

"Infinispan maintains compatibility between the NonBlockingStore interface and the JCache specification by using an adapter for handling blocking code."

Reply:
Sure.